ecs_datadog.tf

```diff
@@ -47,7 +76,7 @@ EOF
 
   volume {
     name      = "cgroup"
-    host_path = "/cgroup/"
+    host_path = "/sys/fs/cgroup/"
```

Comment:
I not sure this location, in some other system, the path might be "/cgroup" ; [link](https://www.datadoghq.com/blog/how-to-collect-docker-metrics/)

@quangnhut123 i think we should introduce new variable for this location and update README for this?